cityofaustin/atd-vz-data
# Patch bugs from missing hasura permissions

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/14786

I've already tested the Editor and Read-only users with these updated permissions so im pretty confident this is good to go :ok_hand: 

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Checkout this branch, apply the metadata to your local database.
2. Test clicking on crash pages while using the Editor user and Read only user.
3. Also test updating the coordinates of a crash as an Editor user, make sure it works and saves the new coordinates.


---
#### Ship list
